### PR TITLE
fix: Spelling and Capitalization

### PR DESCRIPTION
**Pull request by BeastCode**
Most of the fixes are small spelling mistakes. One thing you might want to review is I changed LocalStorage to be pascal case, example:  
LocalstoragePluginOptions  -> LocalStoragePluginOptions
so it's not mixed case.  /Mike

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -401,7 +401,7 @@ export class LoggerPlugin implements NgxsPlugin {
 }
 ```
 
-This pattern allows us to define options while presevering the constructor
+This pattern allows us to define options while persevering the constructor
```
Comment (duanemck) [inline README.md]: I think this should be 'preserving'

Comment (BeastCode) [reply README.md]: Oh, I see what you're saying

Comment (BeastCode) [reply README.md]: Reverted back to the original line and amended the commit.

**Comment by amcdnl** [comment]
No one ever called me a word smith hahaha. Thanks!

**Pull request merged**